application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -111,14 +111,6 @@ public HelpSystemHelper(Config config, Database database, ChatGptService chatGpt
     }
 
     RestAction<Message> sendExplanationMessage(GuildMessageChannel threadChannel) {
-        return mentionGuildSlashCommand(threadChannel.getGuild(), HelpThreadCommand.COMMAND_NAME,
-                HelpThreadCommand.Subcommand.CLOSE.getCommandName())
-                    .flatMap(closeCommandMention -> sendExplanationMessage(threadChannel,
-                            closeCommandMention));
-    }
-
-    private RestAction<Message> sendExplanationMessage(GuildMessageChannel threadChannel,
-            String closeCommandMention) {
         boolean useCodeSyntaxExampleImage = true;
```

Comment:
Do we need this syntax example image anymore? I think issue talked about removing this iirc?

Reply:
done